tests/unit/python/foglamp/services/core/scheduler/test_scheduler.py

```diff
@@ -0,0 +1,1184 @@
+                        "schedule_interval": "00:00:00",
+                        "schedule_time": "",
+                        "schedule_day": "",
+                        "exclusive": "t",
+                        "enabled": "f"
+                    },
+                    {
+                        "id": "81bdf749-8aa0-468e-b229-9ff695668e8c",
+                        "process_name": "North HTTP",
+                        "schedule_name": "sending via HTTP",
+                        "schedule_type": "3",
+                        "schedule_interval": "00:00:30",
+                        "schedule_time": "",
+                        "schedule_day": "",
+                        "exclusive": "t",
+                        "enabled": "f"
+                    },
+                    {
+                        "id": "2b614d26-760f-11e7-b5a5-be2e44b06b34",
+                        "process_name": "North Readings to PI",
+                        "schedule_name": "OMF to PI north",
+                        "schedule_type": "3",
+                        "schedule_interval": "00:00:30",
+                        "schedule_time": "",
+                        "schedule_day": "",
+                        "exclusive": "t",
+                        "enabled": "t"
+                    },
+                    {
+                        "id": "5d7fed92-fb9a-11e7-8c3f-9a214cf093ae",
+                        "process_name": "North Readings to OCS",
+                        "schedule_name": "OMF to OCS north",
+                        "schedule_type": "3",
+                        "schedule_interval": "00:00:30",
+                        "schedule_time": "",
+                        "schedule_day": "",
+                        "exclusive": "t",
+                        "enabled": "f"
+                    },
+                    {
+                        "id": "1d7c327e-7dae-11e7-bb31-be2e44b06b34",
+                        "process_name": "North Statistics to PI",
+                        "schedule_name": "Stats OMF to PI north",
+                        "schedule_type": "3",
+                        "schedule_interval": "00:00:30",
+                        "schedule_time": "",
+                        "schedule_day": "",
+                        "exclusive": "t",
+                        "enabled": "f"
+                    },
+                    {
+                        "id": "543a59ce-a9ca-11e7-abc4-cec278b6b50a",
+                        "process_name": "CC2650POLL",
+                        "schedule_name": "CC2650 poll south",
+                        "schedule_type": "1",
+                        "schedule_interval": "00:00:00",
+                        "schedule_time": "",
+                        "schedule_day": "",
+                        "exclusive": "t",
+                        "enabled": "t"
+                    }
+                ]
+            }
+
+        if table_name == 'scheduled_processes':
+            return {
+                "count": 14,
```

Comment:
Can we make the dict's that are returned global constants without hr count. Then we can add the count in dynamically, to make it easier to maintain. Also we can then have a mechanism to get the count from the constants in the tests rather than hardware the number 14 everywhere.